src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx

```diff
@@ -22,6 +22,14 @@ export const sessionTerm = {
   [SessionTerm.HEALTHY]: t('Healthy'),
   [SessionTerm.ERRORED]: t('Errored'),
   [SessionTerm.UNHANDLED]: t('Unhandled'),
+  duration: t('Session Duration'),
```

Comment:
should name it `sessionDuration`?

Reply:
Would you find it useful even though the whole object is session-oriented?
`sessionTerm.sessionDuration` vs `sessionTem.duration`